README.md

```diff
@@ -13,6 +13,19 @@ fully featured Offline Workflow and Conflict Resolution.
 
 ## Example application
 
-See `sample` for example application. It uses voyager server as its backened (https://github.com/aerogear/voyager-server)
+See `sample` for example application. For now it's using ionic showcase as its backened (https://github.com/aerogear/ionic-showcase)
+
+## How to run the application
+
+- Clone the ionic showcase repository (https://github.com/aerogear/ionic-showcase.git)
+- Run the following commands **to start the server:**
+ 
+  [![Screenshot-2019-06-07-at-11-14-16-PM.png](https://i.postimg.cc/28WFnK0c/Screenshot-2019-06-07-at-11-14-16-PM.png)](https://postimg.cc/N5gHcpqm)
```

Comment:
Antipattern. Never use screenshots for commands

Reply:
Done.